### ISSUE TITLE
RRateLimiter Bug on 3.13.4 : bad argument #2 to 'unpack' (string expected, got nil) 

### ISSUE THREAD

**Issue by XuLeiGithub** [redisson/redisson#5516]
My code
```
@Service
@Slf4j
public class TestIntegration {

    @Resource
    private TestPassApi testPassApi;

    @Resource
    private RedissonClient redissonClient;

    /**
     * limiter
     */
    RRateLimiter testLimiter;

    @PostConstruct
    public void init() {
        testLimiter = redissonClient.getRateLimiter("test-limiter");
        testLimiter.trySetRate(RateType.OVERALL, 1, 1, RateIntervalUnit.SECONDS);
    }

    public Boolean isPass(String testContent) {
        JSONObject jsonObject = new JSONObject();
        jsonObject.put("test_content", testContent);
        RequestBody dto = RequestBody.create(MediaType.get("application/json"), jsonObject.toJSONString());
        Retryer<Boolean> retryer = RetryUtils.getFixedWaitRetryer(1, TimeUnit.SECONDS, 3);
        try {
            return retryer.call(() -> {
                log.info("isPass req: {}", jsonObject.toJSONString());
                // limit 1s 1request
                if (!testLimiter.tryAcquire(1, TimeUnit.MINUTES)) {
                    log.warn("testLimiter limiter timeout");
                    return false;
                }
                var call = testPassApi.test(dto).execute();
                return true;
            });
        } catch (Exception e) {
            log.error("occur error. req: {}", jsonObject.toJSONString(), e);
            return false;
        }
    }
}
```
Error log:
```
Caused by: org.redisson.client.RedisException: ERR Error running script (call to f_df5536206d0c440727a5f17ab38851a31af682c7): @user_script:1: user_script:1: bad argument #2 to 'unpack' (string expected, got nil) . channel: [id: 0x8a39d2f7, L:/127.0.0.1:52098 - R:/127.0.0.1:27374] command: (EVAL), params: [local rate = redis.call('hget', KEYS[1], 'rate');local interval = redis.call('hget', KEYS[1], 'inter..., 5, test-limiter, {test-limiter}:value, {test-limiter}:value:93ad45cb-2a00-4fa7-a689-9a6b589ca1f0, {test-limiter}:permits, {test-limiter}:permits:93ad45cb-2a00-4fa7-a689-9a6b589ca1f0, 1, 1703053565419, 2217768858946301973]

```
I used the tryAcquire method, In theory, when the program cannot get a request, it will wait for at most one minute and then print my warning log.  but in reality, it did not wait and threw an exception directly.

**Comment by mrniko** [comment]
It was fixed in 3.15.3. Can you try it?

**Comment by XuLeiGithub** [comment]
I will try in next version, Thanks.

**Comment by XuLeiGithub** [comment]
> It was fixed in 3.15.3. Can you try it?

I will try in next version, Thanks.
